src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackImagePluginTest.java

```diff
@@ -207,25 +158,17 @@ public void getImageTest() throws FogbowManagerException, UnexpectedException, H
     	Mockito.when(this.properties.getProperty(OpenStackImagePlugin.IMAGE_GLANCEV2_URL_KEY)).thenReturn(imageGlancev2UrlKey);
     	Mockito.when(this.client.doGetRequest(endpoint, this.token)).thenReturn(jsonResponse);
     	
+    	//exercise
     	Image imagePluginOuput = this.plugin.getImage(imageId, this.token);
+    	
+    	//verify
     	Assert.assertEquals(expectedImage, imagePluginOuput);
     }
     
-    private String getImageJsonFromImage(Image image) {
-    	Map<String, String> jsonMap = new HashMap<String, String>();
-    	jsonMap.put(OpenStackImagePlugin.ID_JSON, image.getId());
-    	jsonMap.put(OpenStackImagePlugin.NAME_JSON, image.getName());
-    	jsonMap.put(OpenStackImagePlugin.SIZE_JSON, Long.toString(image.getSize()));
-    	jsonMap.put(OpenStackImagePlugin.MIN_DISK_JSON, Long.toString(image.getMinDisk()));
-    	jsonMap.put(OpenStackImagePlugin.MIN_RAM_JSON, Long.toString(image.getMinRam()));
-    	jsonMap.put(OpenStackImagePlugin.STATUS, image.getStatus());
-    	Gson gson = new Gson();
-    	String jsonResponse = gson.toJson(jsonMap);
-    	return jsonResponse;
-    }
-    
+    //test case: Check if getImage returns null when the state is not ACTIVE_STATE.
     @Test
-    public void testGetImageWhenStateIsNotActivated() throws FogbowManagerException, UnexpectedException, HttpResponseException {
+    public void testGetImageWhenImageStateIsNotActivated() throws FogbowManagerException, UnexpectedException, HttpResponseException {
+    	//set up
```

Comment:
It seems odd we do not have a constant for the it_is_not_activated (or we only need something different than ACTIVE_STATE? In this case, the test is ok)

Reply:
This test checks any other state different from ACTIVE